tests/pass/shims/time-with-isolation.rs

```diff
@@ -0,0 +1,34 @@
+//@compile-flags: -Zmiri-disable-isolation
```

Comment:
The test is called "with isolation" and here you disable isolation... :thinking:

Reply:
Whoops